exporter/collector/integrationtest/metrics_integration_test.go

```diff
@@ -61,6 +64,7 @@ func TestIntegrationMetrics(t *testing.T) {
 		t.Run(test.Name, func(t *testing.T) {
 			test.SkipIfNeeded(t)
 			metrics := test.LoadOTLPMetricsInput(t, startTime, endTime)
+			setSecondProjectInMetrics(t, metrics)
```

Comment:
IIUC, this will override all of the metrics to send to the second project. Does anything get written to the first project anymore?

Reply:
This will override all _existing_ `gcp.project.id` attributes in the input to send to the second project.  Everything that doesn't have a `gcp.project.id` attribute still goes to the first project.

Reply:
I updated the comment on `setSecondProjectInMetrics` and `setSecondProjectInLogs` to make that clearer